.solidarity

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json.schemastore.org/solidaritySchema",
+  "requirements": {
+    "Node": [
+      {
+        "rule": "cli",
+        "binary": "node",
+        "semver": "^10.16.0"
+      }
+    ],
+    "Yarn": [
+      {
+        "rule": "cli",
+        "binary": "yarn",
+        "version": "--version",
+        "semver": "^1.17.3"
+      }
+    ],
+    "Cocoapods and Fastlane": [
+      {
+        "rule": "cli",
+        "binary": "ruby",
+        "semver": "2.5.1",
+        "error": "Ruby is required to run Cocoapods and Fastlane. We recommend installing Ruby with a version manager (https://github.com/echobind/react-native-template-echobind#prerequisites)"
```

Comment:
Can you mention the Ruby version needed? This failed for me, but I had Ruby installed. This way, the developer knows to upgrade/downgrade the Ruby version:
```suggestion
        "error": "Ruby v2.5.1 is required to run Cocoapods and Fastlane. We recommend installing Ruby with a version manager (https://github.com/echobind/react-native-template-echobind#prerequisites)"
```

Reply:
Should I take off the semver here? what version do you have currently?

Reply:
`ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]`